### PR TITLE
create collection success modal

### LINKED ISSUE #8982: Collection creation success modal
next 

after: 
- #8904 

## Design
- updated here: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=6886-20058&mode=design&t=Tx1AQS2qpS8MaBtV-11

image

- primary action - add nfts 
- secondary - view collection

### PR DESCRIPTION
**Pull request by daiagi**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type
- [x] Feature

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #8982 

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)

## Screenshot 📸

- [x] My fix has changed UI
crete collection success modal 09_02_24

### CODE REVIEW

**Comment by prury** [comment]
welcome back 😃

**Comment by prury** [comment]
Looks fine on desktop ✔️ 
I'm having some trouble to test it on mobile, please wait

File: components/create/modals/CollectionSuccessModal.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+<style scoped lang="scss">
+@import '@/assets/styles/abstracts/variables';
+.collection-avatar {
+  @apply bg-background-color;
+}
+
+.border-k-grey-fix {
+  border-color: var(--k-grey-fix);
+}
```
Comment (Jarsen136) [inline components/create/modals/CollectionSuccessModal.vue:97]: How about moving these tailwind class directly on the element?

Comment (daiagi) [reply components/create/modals/CollectionSuccessModal.vue]: thx.
Done

**Comment by daiagi** [comment]
suggested follow ups:
* #9344
* #9345

**Review by Jarsen136 — APPROVED** [review]

**Comment by prury** [comment]
could not test mobile but let's go

**Review by exezbcz — APPROVED** [review]

**Comment by exezbcz** [comment]
thanks!

**Review by preschian — APPROVED** [review]

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 7.06 USD/DOT ~ 7.082 $DOT
🧗 [1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8](https://polkadot.subscan.io/account/1cAsKZYNRb8dkSCpn4eVkEn6ycNZTGoDo5dGDgB8J1UUWK8)
🔗 [0xb4b22c1eab24e425923172ffb952681fd4fd9f487ca71e56883db94036098593](https://polkadot.subscan.io/extrinsic/0xb4b22c1eab24e425923172ffb952681fd4fd9f487ca71e56883db94036098593)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**